book/P1C6_survtsk.qmd

```diff
@@ -10,119 +10,270 @@ abstract: "This chapter introduces the different survival problems and formalise
+However, one can compute $\RMST(100)$ which gives a lower-bound estimate for the expected survival: $\RMST(100) \leq \EE[Y]$.
+This avoids assumptions beyond $\tau=100$ and offers an interpretable prediction: "the average survival time is at least $\RMST(100)$" (@fig-survtsk-rmst).
+The RMST avoids the pitfalls of computing the mean and median by remaining valid even when the predicted distribution is improper, provided that $\tau$ is chosen within the range of observed follow-up times.
+
+As a worked example, say for an individual, $i$, we have: $(t, \hatS_i(t)) = (0, 1), (1, 0.8), (2, 0.4), (3, 0.15)$ then (using a Riemann sum approximation for the integral), $\RMST(4) \approx 1 + 0.8 + 0.4 + 0.15 = 2.35$ whereas $\RMST(3) \approx 1 + 0.8 + 0.4 = 2.2$; the former is a better estimate for $\EE[Y]$.
+
+![Illustrating the RMST at the final observed time-point. The RMST is the area under the survival curve up to the point of interest. The red vertical line represents the final observed time-point, the survival curve (blue) after that point is unknown.](Figures/survtsk/rmst.png){#fig-survtsk-rmst width=80% fig-alt="Graph with 'survival probability S(t)' on the y-axis and 'Time' on the x-axis. A vertical red-line at x=100 represents the cut-off point. The area under the curve before the cut-off point is highlighted in blue, representing the RMST(100)."}
 
 ## Prognostic Index Predictions {#sec-survtsk-PI}
 
-In medical terminology (which is often used in survival analysis), a prognostic index is a tool that predicts outcomes based on risk factors.
+In medical terminology (often used in survival analysis), a prognostic index is a tool that predicts outcomes based on risk factors.
 Given covariates, $\XX \in \Reals^{n \times p}$, and coefficients, $\bsbeta \in \Reals^p$, the *linear predictor* is defined as $\bseta := \XX\bsbeta$.
 Applying some function $g$, which could simply be the identity function $g(x) = x$, yields a *prognostic index*, $g(\bseta)$.
 A prognostic index can serve several purposes, including:
 
-1. Scaling or normalization -- simple functions to scale the linear predictor can better support interpretation and visualisation;
-2. Capturing non-linear effects -- for example the Cox PH model (@sec-models-classical) applies the transformation $g(\bseta) = \exp(\bseta)$ to capture more complex relationships between features and outcomes;
-3. Aiding in interpretability -- in some cases this could simply be $g(\bseta) = -\bseta$ to ensure the 'higher value implies higher risk' interpretation.
+1. Scaling or normalization: simple functions to scale the linear predictor can better support interpretation and visualisation;
+2. Capturing non-linear effects: for example the Cox PH model (@sec-models-classical) applies the transformation $g(\bseta) = \exp(\bseta)$ to capture more complex relationships between features and outcomes;
```

Comment:
Think this could be confusing to some. more complex relationship of features and outcome usually means interactions and non-linearity in feature effects (on the link scale). Here it's simply used as a response function as in a GLM.